GameEngineTests/Agents/BeamAgentTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Linq;
+using GameEngine;
+using GameEngine.Agents;
+using GameEngine.Heuristics;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GameEngineTests.Agents
+{
+    [TestClass]
+    public class BeamAgentTests
```

Comment:
Why is there no unit test for the agent formulating a play?

Reply:
We can certainly check to see if a legitimate play object comes back from the FormulatePlay method, however I don't think there's a good way to verify that any of the beam searches properties are functioning as intended.

Do you want just like a simple check to make sure that *some* play comes back or are you asking for something more complicated?

Reply:
I'm fine with a simple check

Reply:
👍